css/interface.css

```
@@ -12,6 +12,58 @@ body {
   cursor: default;
 }
 
+.restore-disabled {
+  opacity: 0.3;
+}
+
+.btn-round,
+.input-round {
+  border-radius: 20px;
+  outline: none !important;
+}
+
+.btn-round[data-show-alive-source],
+.btn-round[data-show-trash-source] {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.active-source {
+  background-color: #36344c;
+  color: white;
```

Use hexcode as much as you can

```suggestion
  color: #fff;
```